src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -208,6 +208,18 @@ private static void compareFiles(
                 + "_expected_"
                 + outputType
                 + extension);
+
+    // Overwrite expected file with the generated one
+    Boolean rewriteAllExpectedFiles = false;
+    if (rewriteAllExpectedFiles && !outputType.equals("cvr_cdf")) {
+      try {
+        new File(expectedPath).setWritable(true);
+        copyFile(new File(actualOutputPath), new File(expectedPath));
+      } catch (IOException e) {
+        fail(e);
+      }
+    }
```

Comment:
OK, so you're going to delete this block of code.

Reply:
Deleted since the files shouldn't change any more